src/app/game/buildings/buildings.service.ts

```diff
@@ -91,6 +91,12 @@ export class BuildingsService {
     localStorage.setItem('buildings', JSON.stringify(buildings));
   }
 
+  loadFinishedBuildingsFromLS() {
```

Comment:
theese buildings are not finished, so their name should be buildings, loadbuildingsFromLS...